json_container/tests/json_container_test.cc

```diff
@@ -511,6 +511,152 @@ TEST_CASE("JsonContainer::toPrettyString", "[data]") {
     }
 }
 
+TEST_CASE("JsonContainer::toPrettyJson", "[data]") {
+    SECTION("it pretty prints valid json") {
+        SECTION("a null value") {
+            std::string EXPECTED_OUTPUT = "null";
+        
```

Comment:
A bunch of lines with trailing whitespace. Would be nice to fix with `git rebase --whitespace=fix HEAD~1`.